docs/docker-image.md

```diff
@@ -0,0 +1,54 @@
+# `scan-docker-image` (experimental)
+This `vault-radar` command is used for scanning a docker image
+
+## Usage
+
+### Scanning a docker image
+
+Scan a public docker image (or a private image that is already pulled/dowloaded locally) and write the results to a file in CSV format, this is the default format for output. Image reference may optionally include a tag.[Docker engine](https://docs.docker.com/engine/install/) is a pre-requisite for scanning docker images using vault-radar.
+
+```bash
+vault-radar scan-docker-image -i <IMAGE REFERENCE> -o <PATH TO OUTPUT>.csv
+```
+
+### Scanning a private docker image
+
+To scan a private docker image, specify the following environment variables to authenticate against the registry:
+1. `DOCKER_REGISTRY_USERNAME`
+2. `DOCKER_REGISTRY_PASSWORD`
+
+```bash
+vault-radar scan-docker-image -i <IMAGE REFERENCE> -f json -o <PATH TO OUTPUT>.csv
```

Comment:
Can we remove `-f json` since output file has a csv extension?

Reply:
Updated to align with the example